README.md

```diff
@@ -10,14 +10,228 @@ support new services is also a fairly simple process.
+
 
 ## Installation
+
 Just use `pip`
 
-```bash
+```
 pip install lochness
 ```
 
+
+For most recent DPACC-lochness
+
+```
+pip install git+https://github.com/PREDICT-DPACC/lochness
+```
+
+
+For debugging
+
+```
+cd ~
+git clone https://github.com/PREDICT-DPACC/lochness
+pip install -r ~/lochness/requirements.txt
+
+export PATH=${PATH}:~/lochness/scripts  # add to ~/.bashrc
+export PYTHONPATH=${PYTHONPATH}:~/lochness  # add to ~/.bashrc
+```
+
+
+## Running test
+
+- Copy the token template, and add the information for each module.
+
+```
+cd lochness/tests
+cp token_template_for_test_template.csv token_template_for_test.csv
+```
+
+- Run test
+```
+bash run_test.sh 
+```
+
+
+## Setup from a template
+
+### Creating the template
+Setting up lochness from scratch could be slightly confusing in the beginning.
+Try using the `lochness_create_template.py` to create a starting point.
+
+Create an example template to easily structure the lochness system
+
+```sh
+# ProNET
+lochness_create_template.py \
+    --outdir /data/lochness_root \
+    --studies BWH McLean \
+    --sources redcap xnat box mindlamp \
+    --email kevincho@bwh.harvard.edu \
+    --poll_interval 43200 \
+    --ssh_host erisone.partners.org \
+    --ssh_user kc244
+
+# PRESCIENT
+lochness_create_template.py \
+    --outdir /data/lochness_root \
+    --studies BWH McLean \
+    --sources RPMS daris mediaflux mindlamp \
+    --email kevincho@bwh.harvard.edu \
+    --poll_interval 43200 \
+    --ssh_host erisone.partners.org \
+    --ssh_user kc244 
+
+# For more options: lochness_create_template.py -h
+```
+
+
+### Making edits to the template
+
+Running one of the commands above will create the structure below
+
+```sh
+/data/lochness_root/
+├── 1_encrypt_command.sh
+├── 2_sync_command.sh
+├── PHOENIX
+│   ├── GENERAL
+│   │   ├── BWH
+│   │   │   └── BWH_metadata.csv
+│   │   └── McLean
+│   │       └── McLean_metadata.csv
+│   └── PROTECTED
+│       ├── BWH
+│       └── McLean
+├── config.yml
+├── lochness.json
+└── pii_convert.csv
+```
+
+
+
+1. Change information in `config.yml` and `lochness.json` as needed.
+
+
+2. Either manually update the `PHOENIX/GENERAL/*/*_metadata.csv` or
+   amend the field names in REDCap / RPMS sources correctly for lochness to
+   automatically update the metadata files.
+
+   Currently, lochness initializes the metadata using the following field names 
+   in REDCap and RPMS.
+
+- `record_id1`: the record ID field name
+- `Consent`: the field name of the consent date
+- `beiwe_id`: the field name of the BEIWE ID.
+- `xnat_id`: the field name of the XNAT ID.
+- `dropbox_id`: the field name of the Dropbox ID.
+- `box_id`: the field name of the Box ID.
+- `mediaflux_id`: the field name of the Mediaflux ID.
+- `mindlamp_id`: the field name of the Mindlamp ID.
+- `daris_id`: the field name of the DaRIS ID.
+- `rpms_id`: the field name of the RPMS ID.
+
+
+3. Encrypt the `lochness.json` by running
+
+```sh
+bash 1_encrypt_command.sh
```

Comment:
If you can paste two example `1_encrypt_command.sh` and `2_synch_command.sh`, I could give more confident review.

Reply:
Should we mention somewhere that the `.sh` scripts have to be run from `/data/` directory?

Reply:
> If you could paste two example `1_encrypt_command.sh` and `2_synch_command.sh`, I could give more confident review.

The lines below are used to creates the bash scripts, so users can either use or have them as a reference for the next step.
https://github.com/PREDICT-DPACC/lochness/blob/ea34bf10ef5ec901432612b95d53b6a470c53003/scripts/lochness_create_template.py#L90-L106